cloudgarden-pwa/src/components/main/LoginForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import OktaAuth from "@okta/okta-auth-js";
+import { useOktaAuth } from "@okta/okta-react";
+
+interface LoginFormProps {
+  baseUrl: String;
+}
+
+const LoginForm = ({ baseUrl }: LoginFormProps) => {
+  const { authService } = useOktaAuth();
+  const [sessionToken, setSessionToken] = useState();
+  const [username, setUsername] = useState();
+  const [password, setPassword] = useState();
+
+  const handleSubmit = (e: any) => {
+    e.preventDefault();
+    const oktaAuth = new OktaAuth({ issuer: baseUrl });
```

Comment:
in the tutorial this line is 

`const oktaAuth = new OktaAuth({ url: baseUrl });`

but then in the console you get an error saying that you need to pass an issuer to the authSDK

But when I have it like this the error is a generic `AuthApiError`, but it doesn't give much information also a POST request errors out with `net::ERR_NAME_NOT_RESOLVED`

Reply:
I also looked at the github repo to see if it was another situation where stuff was out of date in the tutorial, but it seems fine